exampleSite/config/_default/languages.toml

```diff
@@ -39,6 +41,8 @@ reference = "Meer projecten"
 languageName = "Français"
 contentDir = "content/fr"
 weight = 2
+[nl.params.opengraph]
```

Comment:
Thanks for this PR @d-oit. However, I believe this line should be `[fr.params.opengraph]`?

Reply:
Currently, the Netlify build fails, likely related to the above comment:

```
11:02:23 AM: Error: failed to load config: failed to unmarshal config for path "/opt/build/repo/exampleSite/config/_default/languages.toml": "_stream.toml:1:1": unmarshal failed: toml: table opengraph already exists
```